license-yaml-update.ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
I think this was committed by mistake?

Reply:
:sweat: my bad, must have been not careful during amending.